graylog2-web-interface/src/views/stores/CurrentViewStateStore.test.js

```diff
@@ -100,7 +100,42 @@ describe('CurrentViewStateStore', () => {
     ViewStatesActions.update = updateFn;
     CurrentViewStateStore.onViewStoreChange({ activeQuery: viewId, view: oldViewState });
     CurrentViewStateStore.onViewStatesStoreChange(sMap);
-    CurrentViewStateStore.widgets(expectedWidgets);
+    await CurrentViewStateStore.widgets(expectedWidgets);
+    expect(updateFn).toHaveBeenCalledTimes(1);
+  });
+
+  it('should remove widget positions for deleted widgets', async () => {
+    const widgetOnePos = new WidgetPosition(1, 1, 5, 6);
```

Comment:
Imo the following variable declarations could be changed into one big inline declaration of the view state. We do not reuse these values, so we do not really need to capture them into variables.

Reply:
I have to reuse the widget and widget position for the expected view state. I still prefer the inline declaration and just used `existingViewState.widgets[0]` for the expected state. is this implementation okay or an anti pattern? I can imagine not reusing the "initial state" for the "expected state", to be a best practice.